extensions/ql-vscode/src/sarif-utils.ts

```diff
@@ -1,21 +1,23 @@
 import * as Sarif from 'sarif';
 import * as path from 'path';
-import { LocationStyle, ResolvableLocationValue } from './bqrs-types';
+import { ResolvableLocationValue } from './bqrs-cli-types';
 
 export interface SarifLink {
   dest: number;
   text: string;
 }
 
+interface NoLocation {
```

Comment:
A comment here would be nice, just something like "The type of a result that has no associated location. `hint` is a string intended for display to the user that explains why there is no location."